rust/bitcoin_wallet/README.md

```diff
@@ -2,43 +2,42 @@
 
 ## Summary
 
-This example dapp shows how to build a Bitcoin wallet making use of the [Bitcoin integration](https://smartcontracts.org/docs/developers-guide/concepts/bitcoin-integration.html).
-
-This dapp is under active development and not ready to be used yet.
-<!--Once some parts of its functionality are implemented, this README will be updated accordingly with instructions on how to build and run the dapp.-->
+This example dapp shows how to build a basic Bitcoin wallet making use of the
+Internet Computer's [Bitcoin integration](https://smartcontracts.org/docs/developers-guide/concepts/bitcoin-integration.html).
 
 ## Step-by-step tutorial
 
-Install required node modules for the Bitcoin wallet webapp.
+Install the required node modules for the Bitcoin wallet webapp:
 
 ```bash
 npm install
 ```
 
 While working on the Internet Computer does not require more configuration, working locally does. The additional instructions are provided in [the next section](#testing-locally).
 
-To deploy the development Internet Identity canister and the Bitcoin wallet canister locally.
+Run the following commands to deploy the development Internet Identity canister and the Bitcoin wallet canister locally:
 
 ```bash
 dfx deploy internet_identity --argument '(null)'
 dfx deploy bitcoin_wallet_assets
 ```
 
-To deploy the Bitcoin wallet canister on the Internet Computer.
+Run the following command to deploy the Bitcoin wallet canister on the Internet Computer:
 
 ```bash
 II_CANISTER_ID=identity dfx deploy --network ic bitcoin_wallet_assets
 ```
 
 ## Testing locally
 
-The Internet Computer Bitcoin Library invokes the Bitcoin integration API through the management canister. In order to test the Bitcoin wallet locally, you have to follow the following instructions.
+The Bitcoin wallet invokes the Bitcoin integration API through the management canister.
+In order to test the Bitcoin wallet locally, follow the instructions below.
 
 ### Prerequisites
 
 - [Bitcoin Core](https://bitcoin.org/en/download). Mac users are recommended to download the `.tar.gz` version.
 
-The first step would be to setup a local Bitcoin network.
+The first step is to setup a local Bitcoin network.
```

Comment:
Minor thing, as the _To deploy the ..._ sentences, are also part of the Bitcoin developer preview [1](https://raw.githubusercontent.com/dfinity/bitcoin-developer-preview/8ba02e33a5c36ca7b279ba047ac07af3871ae2ce/README.adoc#L82) and [2](https://raw.githubusercontent.com/dfinity/bitcoin-developer-preview/8ba02e33a5c36ca7b279ba047ac07af3871ae2ce/examples/README.adoc#L20-L28).

I modified in the incoming PR the Bitcoin library accordingly.